<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1795</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>1731</width>
     <height>521</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <column>
    <property name="text">
     <string>타임스탬프</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>폐 정수기 수거 요청</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>비고(직원메모)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>접수경로</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>제품사</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>주문 상태</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>설치 요청일</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>모델명</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>렌탈료</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>색상</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>패지키</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>계약자</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>연락처</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>생년월일</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>설치주소</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>조리수 벨브</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>폐 정수기 수거 요청</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>지불 방법</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>은행/카드사</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>계좌/카드번호</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>유효기간/예금주</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>최종 수정인</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>최초 작성자</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="win_modify_complete">
   <property name="geometry">
    <rect>
     <x>1570</x>
     <y>40</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>등록 완료</string>
   </property>
  </widget>
  <widget class="QPushButton" name="win_modify_standby">
   <property name="geometry">
    <rect>
     <x>1350</x>
     <y>40</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>등록 대기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="win_data_modify">
   <property name="geometry">
    <rect>
     <x>1530</x>
     <y>660</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>데이터 수정</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="win_enroll_man">
   <property name="geometry">
    <rect>
     <x>1350</x>
     <y>700</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>670</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>전산 등록인</string>
   </property>
  </widget>
  <widget class="QPushButton" name="win_find_btn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>등록 대기 조회</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>모두</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LG</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SK</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HYUNDAI</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CUCKOO</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CHUNGHO</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>KYOWON</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BODYFRIEND</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RUHENS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>WELRIX</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NOVITA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SAMSUNG</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="win_reservation_btn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>40</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>예약 조회</string>
   </property>
  </widget>
  <widget class="QComboBox" name="win_num_of_find_combobox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>10000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30000</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>조회 갯수</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="win_modify_reservation">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>40</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>예약</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>660</y>
     <width>381</width>
     <height>91</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>700</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>오류창</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
